docs/job-scheduling.md

```diff
@@ -252,10 +252,11 @@ properties:
 
 The pool properties can be set by creating an XML file, similar to `conf/fairscheduler.xml.template`,
 and either putting a file named `fairscheduler.xml` on the classpath, or setting `spark.scheduler.allocation.file` property in your
-[SparkConf](configuration.html#spark-properties).
+[SparkConf](configuration.html#spark-properties). The file path can either be a local file path or HDFS file path.
```

Comment:
Actually, this line isn't completely true. It can be a local file path only when `fs.default.scheme` is `file:///` which usually isn't in production.

So, if users from old Spark versions use a path like `/path/to/file`, the files will be written into HDFS after the upgrade.

Can we at least update the migration guide? We should also mention that it respects Hadoop properties now.

Reply:
+1 for @HyukjinKwon 's advice.

Reply:
@HyukjinKwon sorry, not sure I get your point.

> So, if users from old Spark versions use a path like /path/to/file, the files will be written into HDFS after the upgrade.

Why we need to write files into HDFS ? This PR is to support read remote file as the schedule pool. I think there is no behavior change but just a new feature.

Reply:
I see , if user specify a path `/path/to/file` with no scheme then after this PR, Spark will load it from HDFS instead of local. That worth a migration guide , thank you point out this ! @HyukjinKwon @dongjoon-hyun

Reply:
thanks man!